docs/development/environment.md

```diff
@@ -0,0 +1,7 @@
+# Developer Environment
+
+A devcontainer is provided to ease the development of the runner manager.
+It can be used with GitHub's Codespaces or Visual Studio Code.
+
+It is to be considered as the only supported development environment
+for ease of support and reproducibility.
```

Comment:
There will be no chart or kustomize ?

Reply:
Not for a developer environment, at least not for now. 

For the service's deployment it's another story.